model/registry/incident.yaml

```diff
@@ -0,0 +1,25 @@
+groups:
+  - id: registry.incident
+    prefix: incident
+    type: attribute_group
+    brief: >
+     This group describes attributes specific to incidents.
+    attributes:
+      - id: name
```

Comment:
Is this intended to be a human-readable name for the incident (a "display name" if you will)? Or is this intended to be the identity (e.g. "REST resource name") of the incident.

It is unclear from the description and the example.